autoload/sy.vim

```diff
@@ -147,3 +142,31 @@ endfunction
 function! sy#buffer_is_active()
   return exists('b:sy') && b:sy.active
 endfunction
+
+function! s:skip(path)
+  if &diff || !filereadable(a:path)
+    return 1
+  endif
+
+  if exists('g:signify_skip_filetype')
+    if has_key(g:signify_skip_filetype, &filetype)
+      return 1
+    elseif has_key(g:signify_skip_filetype, 'help') && (&buftype == 'help')
+      return 1
+    endif
+  endif
+
+  if exists('g:signify_skip_filename') && has_key(g:signify_skip_filename, a:path)
+    return 1
+  endif
+
+  if exists('g:startify_skip_filename_pattern')
```

Comment:
`g:startify_skip_filename_pattern` -- wrong project. :) Same with the commit message.